### PR TITLE
Anil/fix staking

### PR DESCRIPTION
**Pull request by anilCSE**

### CODE REVIEW

File: x/staking/keeper/msg_server.go
Comment (anilCSE) [inline x/staking/keeper/msg_server.go:167]: @murthyvitwit update this too

File: x/staking/client/cli/tx.go
Comment (anilCSE) [inline x/staking/client/cli/tx.go:278]: @murthyvitwit let's fix this too before merging

**Review by anilCSE — COMMENTED** [review]
Lgtm, pending tests pass

**Review by jdkanani — APPROVED** [review]

**Pull request merged**
